generators/server/templates/src/main/java/package/domain/PersistentAuditEvent.java.ejs

```diff
@@ -143,6 +148,9 @@ public class PersistentAuditEvent implements Serializable {
     @Transient
         <%_ } _%>
     <%_ } _%>
+    <%_ if (databaseType === 'neo4j') { _%>
+    @Transient
```

Comment:
Right now mapping of maps is not part of SDN/RX (on purpose). So let's make it transient for now. As we plan to remove the audit feature (and neo is still in beta) I would say thats okay.

Reply:
This fixes also the daily build as before the test was not generated